lambda_functions/delivery_status_processor_lambda/delivery_status_processor_lambda.py

```diff
@@ -50,6 +96,15 @@ def delivery_status_processor_lambda_handler(event: any, context: any):
 
         logger.info("Valid ALB request received")
         logger.debug(event["body"])
+        if "TwilioProxy" in event["headers"]["user-agent"] \
+                and context \
+                and not validate_twilio_event(event):
+            logger.info("Returning 403 on unauthenticated Twilio request")
```

Comment:
Please review the [logging doc](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/logging_policy.md) and change to the appropriate log level.